### DISCUSSION TITLE
Can Bracket be used to track ELO score of multiple players?

### DISCUSSION THREAD

**Discussion by arthurbenemann** [evroon/bracket#799] [Q&A] [answered]
Can the tool be used to track ELO score of multiple players over the course of friendly matches only? Alternatively a tournament with infinite matches?

**Comment by henninfi** [comment]
I suppose you can in theory, but as matches in bracket are tied to tournaments, you need to create new stages or so to be able to report new matches. If you're keen to keep track of matches outside of tournaments feel free to test my free app, https://rankit.no .

**Comment by evroon** [comment, marked as answer]
The Swiss stage item in bracket allows you to do this, you can just keep adding an infinite amount of rounds with matches and keep track of the scores.

For now, it gives you a list of balanced matches with teams of similar ELO score. So you can't add any match between two arbitrary opponents for now, they have to be of similar ELO. I don't know if that's a limitation for you.

**Reply by arthurbenemann** [reply]
@evroon My use case is much simpler than a full tournament, and we have a wide range of skills. It doesn't match the intent of the project.

Thank you for the follow-up.

**Reply by arthurbenemann** [reply]
@evroon ended-up writing a very basic version, that uses a Gsheets spreadsheet as the database. Made it easy to share around the office. https://github.com/arthurbenemann/pongtracker

GGs

**Reply by henninfi** [reply]
Cool project! Can I ask you if you tried rankit.no, giving free elo-ladder just for this purpose. You can add players without them having to register, or if they want to be able to log in to register matches or check rank they can register themselves. If you're not keen your feedback would be very much appreciated!
